packages/kotti-ui/source/kotti-card/types.ts

```diff
@@ -0,0 +1,18 @@
+import { z } from 'zod'
+
+export namespace KottiCard {
+	export enum ImagePosition {
+		BOTTOM = 'bottom',
+		MIDDLE = 'middle',
+		TOP = 'top',
+	}
+	export const imagePositionSchema = z.nativeEnum(ImagePosition)
+
+	export const propsSchema = z.object({
+		imgPosition: imagePositionSchema.default(ImagePosition.TOP),
+		imgUrl: z.string().default(''),
```

Comment:
`imgUrl: z.string().nullable().default(null)`

wouldn't change anything else internally because we guard with `v-if` which will Boolean the `''`

but we follow the `null` for null primitives as you know already, so might as well make the change